sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -736,13 +736,15 @@ class SparkSession private(
   }
   // scalastyle:on
 
+  def stop(): Unit = stop(0)
+
   /**
    * Stop the underlying `SparkContext`.
    *
    * @since 2.0.0
    */
-  def stop(): Unit = {
-    sparkContext.stop()
+  def stop(exitCode: Int = 0): Unit = {
```

Comment:
@AngersZhuuuu this is a breaking change. You will need to define two stop methods, one with the status code, and one without.